modules/test/playwright/helpers/ObjectAdminApiHelper.ts

```diff
@@ -46,6 +46,16 @@ export class ObjectAdminApiHelper {
 		);
 	}
 
+	async postObjectFieldByExternalReferenceCode(
```

Comment:
Hey @rafaellajordao, could you sort this method?

1. postObjectActionByExternalReferenceCode
2. postObjectDefinition
3. postObjectFieldByExternalReferenceCode
...

Reply:
sure! fixed!